Pod/Classes/MenuView.swift

```diff
@@ -8,6 +8,16 @@
 
 import UIKit
 
+public struct MultilineMenuItem: MenuItemType {
```

Comment:
At least, this PR is about multiline text menu. Please create a new PR of menu view only usage.
I'll consider it at that time.